### ISSUE TITLE
Converting unsupported operations -  Enter, Exit, Merge, Switch

### ISSUE THREAD

**Issue by abdullah-alghamdi** [tensorflow/tensorflow#27832]
**System information**
- OS Platform and Distribution : Ubuntu 16.04.3
- TensorFlow installed from: pip
- TensorFlow version: 2.0 alpha

I am using bazel to convert .pb file to .tflite. This is the command that I used to convert the model. 

abdullah@abdullah-OptiPlex-7060:~/tensorflow/tensorflow/lite/toco$ bazel run . --define=with_select_tf_ops=true -- --output_file=/home/abdullah/Documents/CRNN_MODEL/bazelled.tflite   --input_file=/home/abdullah/Documents/CRNN_MODEL/my_model__.pb   --input_arrays=the_input   --output_arrays=softmax/truediv 

I receive the following output: 
```
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: ADD, CONCATENATION, CONV_2D, DIV, EXP, EXPAND_DIMS, FULLY_CONNECTED, LESS, LOGICAL_AND, MAXIMUM, MAX_POOL_2D, MINIMUM, MUL, PACK, RANGE, REDUCE_MAX, RESHAPE, REVERSE_V2, SHAPE, STRIDED_SLICE, SUB, SUM, TANH, TILE, TRANSPOSE, UNPACK, ZEROS_LIKE. Here is a list of operators for which you will need custom implementations: Enter, Exit, LoopCond, Merge, Switch, TensorArrayGatherV3, TensorArrayReadV3, TensorArrayScatterV3, TensorArraySizeV3, TensorArrayV3, TensorArrayWriteV3
```
When I added the  --allow_custom_ops flag to the same command above, I received the following output:
``` 
E tensorflow/lite/toco/toco_tooling.cc:456] TensorFlow Lite currently doesn't support control flow ops: Enter, Exit, Merge, Switch.
```

It is possible to avoid using those control flow ops (Enter, Exit, Merge, Switch) in my original .pb model? 
Thanks for your help.

**Comment by ymodak** [comment]
Duplicate #27241
